testing/tests/DevExpress.ui.widgets/toast.tests.js

```diff
@@ -152,6 +152,29 @@ QUnit.module('general', moduleConfig, () => {
 
         assert.equal($('.dx-toast-content').filter(':visible').length, 2, 'both toasts are visible');
     });
+
+    QUnit.test('toast should repeat trying to close after hiding is cancelled (T1156504)', function(assert) {
+        let hidingCallCount = 0;
+        const displayTime = 500;
+        this.instance.option({
+            displayTime,
+            onHiding: (e) => {
+                if(hidingCallCount === 0) {
+                    e.cancel = true;
+                }
+                ++hidingCallCount;
```

Comment:
Let's not use unary increment here https://github.com/airbnb/javascript#variables--unary-increment-decrement